### PR TITLE
Added Fonts to Theme Using TWColors Util

### PR DESCRIPTION
**Pull request by eprisr**

### CODE REVIEW

File: src/utils/index.ts [hunk 1]
```diff
@@ -0,0 +1,289 @@
+// From tw-colors: https://github.com/L-Blondy/tw-colors/blob/master/lib/index.ts
+
```
Comment (michael-gokey-architect) [inline src/utils/index.ts:2]: @eprisr in your next go round with clean up, can you move this from utils, to Core/utils

File: tailwind.config.js [hunk 2]
```diff
@@ -1,5 +1,7 @@
 /** @type {import('tailwindcss').Config} */
+const plugin = require("tailwindcss/plugin");
 const { createThemes } = require('tw-colors');
+import { createThemeFonts } from './src/utils/index';
```
Comment (michael-gokey-architect) [inline tailwind.config.js:4]: 👍

File: tailwind.config.js [hunk 3]
```diff
@@ -72,24 +64,23 @@ module.exports = {
         strict: true,
       }
     ),
+    createThemeFonts({
```
Comment (michael-gokey-architect) [inline tailwind.config.js:67]: ❤

File: tailwind.config.js [hunk 4]
```diff
@@ -72,24 +64,23 @@ module.exports = {
...
+      forest: {
+        body: '"Tenor Sans", sans-serif',
+        display: '"Merienda", cursive',
+      },
+      racer: {
+        body: "Montserrat",
+        display: "Anton",
+      },
+      "retro-diner": {
```
Comment (michael-gokey-architect) [inline tailwind.config.js:76]: @eprisr I love seeing this come to action.
In your next round, can you add PatchWrk, NYIAS, DragonCon, Grey, and set grey as default, 
keeping Forest, Race, RetroDiner in there.

**Review by michael-gokey-architect — APPROVED** [review]

File: src/utils/index.ts (hunk 1 above)
Comment (eprisr) [reply src/utils/index.ts:2]: Missed this memo. I need to get in the habit of checking my notifications after a PR.

**Pull request merged**